gridstatus/caiso.py

```diff
@@ -23,6 +22,14 @@
 from gridstatus.decorators import support_date_range
 from gridstatus.gs_logging import log
 from gridstatus.lmp_config import lmp_config
+from gridstatus.utils import (
```

Comment:
why this change?

I don't feel strongly, but I like when I'm reading the code later in the file to be able to see that im using a repo-wide `util` function. 

also, this change broke the code: https://github.com/gridstatus/gridstatus/actions/runs/11113242953/job/30877128940?pr=442#step:7:31

Reply:
yes saw that. Change was because the other imports were by function and that tends to be my preference, but will alter it back to avoid the code break.

Reply:
done.